tutorials/scripting/gdscript/warnings/standalone_expression.rst

```diff
@@ -0,0 +1,51 @@
+STANDALONE_EXPRESSION
+=========================
+
+The warning message is:
+
+.. code-block:: none
+
+    Standalone expression (the line may have no effect).
+
+The default warning level for this warning is **Warn**.
+To modify it, see :ref:`ProjectSettings.debug/gdscript/warnings/standalone_expression<class_ProjectSettings_property_debug/gdscript/warnings/standalone_expression>`.
+
+When this warning occurs
+------------------------
+
+This warning may appear when writing an expression (a combination of values) that aren't assigned to anything or passed to a function:
```

Comment:
Not necessarily true. `+++++1` is also an expression.
```suggestion
This warning may appear when writing an expression that is not assigned to anything or passed to a function:
```

Reply:
Good point. Perhaps we don't need to try to explicitly define an expression here. Maybe there's somewhere else in the documentation that describes expressions, and we can simply link a reference to that, in case someone's confused about what the warning means by an "expression"?

Reply:
I think expressions themselves are something you intrinsically learn about when programming. I don't think it should be explained here, but I guess it wouldn't hurt to link https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html#expressions